init

```diff
@@ -2,17 +2,25 @@
 
 echo "-------Beginning bootstrap script-------"
 
+if [ -z "$1" ]; then
+  VERSION="v2"
+else
+  VERSION="$1"
```

Comment:
I did it this way so we can customize the bootstrap command in the CF params to run v3 if needed. But etcd could work too